Task4_Battleships/Board/GameBoard.cs

```diff
@@ -31,21 +35,38 @@ public void PrintWholeBoard()
         private void PrintSide(List<BoardSquare> side)
         {
             Console.Write("   ");
-            foreach (string letter in Enum.GetNames(typeof(CoordinateLiteral))) 
+            foreach (string letter in Enum.GetNames(typeof(CoordinateLiteral))) // prints the letter coordinate frame
             {
+                Console.ForegroundColor = ConsoleColor.Cyan;
                 Console.Write(letter + " ");
+                Console.ForegroundColor = ConsoleColor.White;
             }
             Console.WriteLine();
+            int numberFrameIdx = 1;
             foreach (BoardSquare square in side)
             {
-                if (square.Coordinate.Item1 == 0 && square.Coordinate.Item2 == 0)
-                    Console.Write(" ");
-                else if (square.Coordinate.Item1 == 0)
+                if (square.Coordinate.Item1 == 0 && square.Coordinate.Item2 == 0) // prints the number coordinate frame
                 {
-                    if (square.Coordinate.Item2 == WIDTH)
-                        Console.Write(square.Coordinate.Item2 + " ");
+                    if (numberFrameIdx > 9)
```

Comment:
9 ?

Reply:
Checking for double digit numbers purely for visual elements.